CARTOGRAPHY.md

```diff
@@ -1,39 +1,49 @@
+## Main goals
 
-Colour definitions should, where useful, be put into variables at the top of the file, to enable easier customisation.
+The following goals form an ordered list with the most important on top.  This 
+essentially means pursuit of the goals further at the bottom should not 
+sacrifice the goals higher up.  All goals are however relevant to fulfill the 
+purposes above and should be balanced as far as possible.  Some compromises 
+need to be made with the higher priority goals on occasions to not completely 
+ruin the lower priority ones.  Apart from these goals there are of course also 
+technical constraints and requirements that need to be taken into account.
 
-## Data manipulation
+1. **Legibility** - The map should be intuitively readable by users with some general experience using maps without a map key.  A map key or more extensive experience using this map style can be required for clearly identifying minor differences or the exact meaning of certain features but in broad strokes orientation and identification of map elements should be possible on an intuitive level.
 
-OpenStreetMap data has to be manipulated for rendering, but since this style is intended for use by mappers to check their work, it should minimise any distortions. For example, line-smoothing improves the look of railways and rivers, but introduces confusion for mappers. Post processing of geometries can improve the cartographic results, but breaks the cause-and-effect between editing the data and seeing the results.
+2. **Being understandable and supportive for the mappers** - To serve as feedback for mappers and encourage correct mapping this style needs to render the data in a way that allows mappers to understand how the data produces a certain rendering result based on basic observation without in depth understanding how map rendering works or looking at the style implementation.
 
-For similar reasons, use of external non-OSM data should be avoided.
+3. **Diversity** - The style should represent the diversity of the OSM community and geography in general.  The most obvious element to serve this goal is showing the local names everywhere on earth in their respective scripts.  This goal however goes beyond labels.  Both physical and cultural geography differs a lot globally and the aim is to represent this variety with equal determination - well mapped areas are not supposed to have more weight here than less mapped parts of the world.  This also means the target map user is the potential global map user and no special consideration is given to the current geographic distribution of actual map use.
 
-## Legibility
+4. **Clarity** - This is strongly and directly related to legibility but more specifically means the map should be readable with as little effort as possible and be pleasant to look at both with high and low levels of interest and concentration.
```

Comment:
Intuitively does not mean easily - it just means you don't need additional information to figure it out, it might still take a lot of effort.

In general i think we should do this based on suggestions.  If you think something should be changed make a suggestion to change it.  This way we'd make better progress than by arguing about the meaning of words.